styles.css

```diff
@@ -87,3 +87,243 @@ main ul {
 .frontpage {
   padding: 0 5%;
 }
+
+/* Works */
+
+.works {
+  display: grid;
+  justify-items: center;
+  margin-bottom: 30%;
+}
+
+.projimage {
+  display: flex;
+  justify-content: center;
+}
+
+.recent {
+  color: #1c1a19;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  padding-top: 25%;
+  border-bottom: 1px solid #212121;
+  padding-bottom: 10%;
+  margin-bottom: 15%;
+}
+
+.card {
+  padding-bottom: 15%;
+  margin: auto;
+  width: 90%;
+}
```

Comment:
- [ ] **As suggested in the project requirements** The content of a single project should be aligned with the `flexbox` 😎, aligning the content of a project with `flexbox` allows for more efficient and flexible web page design, improving the overall user experience across different devices and screen sizes. 👍

Reply:
@TuyishimireEric Please give clarity on your review in this section. Thank you